src/popup/components/Tokens.vue

```diff
@@ -46,6 +47,8 @@
 import AeIcon from '../../icons/tokens/ae.svg';
 import { AETERNITY_CONTRACT_ID, AETERNITY_SYMBOL } from '../utils/constants';
 
+const SIZES = ['rg', 'md', 'xl']; // TBD - add more sizes to icons
```

Comment:
In my opinion there is no need to add the "TBD" comment here. And If you feel that we really need it I suggest to use "TODO:" pattern as described here: https://www.jetbrains.com/help/webstorm/using-todo.html#default_patterns
It allows to scrape the todos and some IDEs/addons color the todo comments in red if written this way.